### ISSUE TITLE
queue gives an error module in NetMHCcons

### ISSUE THREAD

**Issue by arahuja** [openvax/mhctools#21]
Using NetMHCcons in the readme example

```
from mhctools import NetMHCcons

predictor = NetMHCcons(alleles=["A*02:01", "hla-a0101"])

# scan the short proteins 1L2Y and 1L3Y for epitopes
protein_sequences = {
  "1L2Y": "NLYIQWLKDGGPSSGRPPPS",
  "1L3Y": "ECDTINCERYNGQVCGGPGRGLCFCGKCRCHPGFEGSACQA"
}

epitope_collection = predictor.predict(protein_sequences)
```

gives 

```
TypeError                                 Traceback (most recent call last)
<ipython-input-5-2d10a5c67892> in <module>()
----> 1 epitope_collection = predictor.predict(protein_sequences)

/demeter/users/ahujaa01/src/hammerlab/mhctools/mhctools/netmhc_cons.pyc in predict(self, fasta_dictionary)
     92             run_multiple_commands_redirect_stdout(
     93                 commands, print_commands=True,
---> 94                 process_limit=self.process_limit)
     95             for output_file, command in commands.iteritems():
     96                 # closing/opening looks insane

/demeter/users/ahujaa01/src/hammerlab/mhctools/mhctools/process_helpers.pyc in run_multiple_commands_redirect_stdout(multiple_args_dict, print_commands, process_limit, polling_freq, **kwargs)
    104     assert all(hasattr(f, 'name') for f in multiple_args_dict.keys())
    105     start_time = time.time()
--> 106     processes = queue(maxsize=process_limit)
    107
    108     def add_to_queue(process):

TypeError: 'module' object is not callable
```

**Comment by arahuja** [comment]
This is on python 2, by the way.

I get the following error on python 3

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/arahuja/src/hammerlab/mhctools/mhctools/netmhc_cons.py", line 52, in predict
    fasta_dictionary, max_file_records=self.max_file_records)
  File "/Users/arahuja/src/hammerlab/mhctools/mhctools/file_formats.py", line 42, in create_input_fasta_files
    "w_%s" % i, prefix="peptide", delete=False) for i in i_range]
  File "/Users/arahuja/src/hammerlab/mhctools/mhctools/file_formats.py", line 42, in <listcomp>
    "w_%s" % i, prefix="peptide", delete=False) for i in i_range]
  File "/Users/arahuja/anaconda/envs/py34/lib/python3.4/tempfile.py", line 463, in NamedTemporaryFile
    newline=newline, encoding=encoding)
ValueError: invalid mode: 'w_0'
```

**Comment by tavinathanson** [comment]
The first issue is addressed in #22

The second seems separate, so I'll leave this open for now.
